packages/tools/client-logger/app-insights-logger/src/test/ClientUtilities.ts

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ConnectionState } from "@fluidframework/container-loader";
+import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import {
+	TinyliciousClient,
+	TinyliciousContainerServices,
+} from "@fluidframework/tinylicious-client";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { FluidAppInsightsLogger } from "../fluidAppInsightsLogger";
+// import { FluidAppInsightsLogger } from "../fluidAppInsightsLogger";
+
+/**
+ * This module contains Fluid Client utilities, including Container creation / loading.
+ */
+
+/**
+ * Type returned from when creating / loading the Container.
+ */
+export interface ContainerLoadResult {
+	container: IFluidContainer;
+	services: TinyliciousContainerServices;
+}
+
+/**
+ * Basic information about the container, as well as the associated audience.
+ */
+export interface ContainerInfo {
+	/**
+	 * {@link ContainerInfo.container}'s unique ID.
+	 */
+	containerId: string;
+
+	/**
+	 * The initialized Fluid Container.
+	 */
+	container: IFluidContainer;
+
+	/**
+	 * Optional nickname for the Container to be used in the debugger and associated UI.
+	 */
+	containerNickname?: string;
+}
+
+function initializeTinyliciousClient(): TinyliciousClient {
+	console.log(`Initializing Tinylicious client on port ${process.env.PORT}...`);
+	const appInsightsClient = new ApplicationInsights({
+		config: {
+			// connectionString: ""
+		},
+	});
+
+	return new TinyliciousClient({
+		logger: new FluidAppInsightsLogger(appInsightsClient),
```

Comment:
I was thinking about this a bit. One of the goals of our logger interface was for it to be as simple as possible, so it could be trivially adapted to. The change is this PR boils down to basically this. Is it worth even having a package, and all that comes with it (support, maintenance, etc). Or would an example be enough?
```suggestion
		logger: {
		    send(event: ITelemetryBaseEvent): void {
		        appInsightsClient.trackEvent({
			    name: event.eventName,
			    properties: event,
		    });
		},
```

Reply:
Agreed with @alexvy86, the packaged solution need to provide end to end functionality with the developer only needing to "flip switch" enable App Insights